pkg/neonvm/controllers/vm_controller.go

```diff
@@ -1342,10 +1342,16 @@ func podSpec(
 						if config.DisableRunnerCgroup {
 							cmd = append(cmd, "-skip-cgroup-management")
 						}
+
+						memhpAutoMovableRatio := config.MemhpAutoMovableRatio
+						if specValue := vm.Spec.Guest.MemhpAutoMovableRatio; specValue != nil {
+							memhpAutoMovableRatio = *specValue
+						}
```

Comment:
Worth using `lo.FromPtrOr` ? (from `github.com/samber/lo`)

Up to you

Reply:
Tried it, the line was too long, decided not to use it.